web/src/modules/layout/components/__tests__/HeaderLogo.spec.js

```diff
@@ -16,49 +16,47 @@ describe('HeaderLogo', () => {
     startDate: '2021-03-08T00:00:00.000Z',
     endDate: '2021-03-15T00:00:00.000Z'
   }
-  let config
+  const previousConfig = buildConfig()
+  let config = previousConfig
   let clock
 
-  beforeEach(() => {
-    config = buildConfig()
-  })
-
   afterEach(() => {
     clock.uninstall()
+    config = previousConfig
   })
 
   it('should show the regular header app icon if there is no campaign', () => {
     clock = lolex.install({ now: 1615374110000, toFake: [] }) // Wed Mar 10 2021 11:01:50 GMT+0000
     config.campaign = undefined
-    config.icons.appLogo = 'my-regular-logo'
-    const { getByAltText } = render(<HeaderLogo link='https://som.el.ink/tomy/home/page' buildConfig={config} />)
+    config.icons.appLogo = '/my-regular-logo'
+    const { getByAltText } = render(<HeaderLogo link='https://som.el.ink/tomy/home/page' />)
```

Comment:
I'd recommend to use the domain example.com to prevent any "accidents", i.e. dataleaks to not trusted domain owners.